services/api/emails/layout.html

```diff
@@ -0,0 +1,353 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="x-apple-disable-message-reformatting" />
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <meta name="color-scheme" content="light dark" />
+    <meta name="supported-color-schemes" content="light dark" />
+    <title></title>
+    <style type="text/css" rel="stylesheet" media="all">
+      /* Base ------------------------------ */
+
+      @import url('https://fonts.googleapis.com/css?family=Nunito+Sans:400,700&display=swap');
```

Comment:
not sure we were maybe doing this before but do we really have to have a custom font in our emails? I so much prefer not to stand out and look more or less like the client... in a typographic sense... I mean it's not like it's a website but more to the point bedrock-core doesn't really have an out of the box "brand" it's trying to apply.

Reply:
Yeah i think you are right, that would be good.
Let me fix up the front and make sure the primary color is the same